### ISSUE TITLE
Other players sometimes get stuck in spinning animation with propellor mushroom. 

### ISSUE THREAD

**Issue by Proxofer** [ipodtouch0218/NSMB-MarioVsLuigi#95]
i seen this bug multiple times by now, the player goes spin flying, but when he touches the ground and walks, the character keeps the idle spin animation (although im pretty sure the animation speeds up with the velocity speed of the player's character). this bug does not always pop up, i only seen this 3 times in different lobby's while spectating.

**Comment by Proxofer** [comment]
i've been thinking if this could also be caused by outdated graphics drivers, due to mine being last updated in 2015

**Comment by ipodtouch0218** [comment]
Nah, it's caused by a desync. Planned to be fixed in v1.8.0.0-beta
